other-docs/guides/getting-help-with-altis.md

```diff
@@ -0,0 +1,50 @@
+
+We break down support into different types. Each has their own language and processes, but they're somewhat intuitive once you internalize the language a little. Let's go over the types.
+
+<img width="1282" alt="Altis Dashboard Create Support Ticket page, Question selected in Type dropdown menu" src="https://user-images.githubusercontent.com/1263370/113014673-b2630600-9174-11eb-88ae-ed3cc6b98078.png">
+
+We currently have four types of support. Below is a description of each and how they are different from each other.
+
+# Incident
+
+Incidents are unplanned events that negatively impact service, or quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if something isn’t getting the expected result, this would be an incident, and requires investigation.
+
+From the support side, Incidents are the things that make us scamble to respond and jump on the problem. If the site is down, this is an Incident, with a prioriy of ‘Urgent’. Alarm bells will ring, and we’ll put out the fires to get things running again by any means neccessary.
+
+Incidents focus on immediate attention, with an emphisis on immediate remediation. Remediating an Incident can take the form of temporary patches, which brings us to the next support type.
+
+# Problem
+
+Problems are the cause of, or potential cause of an Incidents. Incidents revolve around something the unexpected. If a site is down, that’s an Incident, but the Problem is a known cause of an issue you wish to track, and coordinate help with, problems typically take longer to resolve, as an appropriate solution is thoughtout, and worked on.
+
+# Task 
+
+There’s an array of tasks that might you need help with. Typically, this might mean provisioning of infrastructure to accommodate some need, it might be providing access to some tool or service used by the team, or it could be to run an operation that requires a cloud engineer. Tasks should be well described.
+
+# Question (Service Requests)
+
+This type is fairly self explanatory. Have a question for us? Ask!
+
+If you need some kind of consultancy, technical question, clarification on a piece of the product, just open a support question.
+
+# Other methods of support
+
+Mentioned at the top of this page were two main ways of receiving Altis support, covered thus far is just the primary method. The second method is email, which is typically discouraged, but it’s useful to have this ability when needed.
+
+We have a few email addresses linked to our support so we can track issues that come in. Those are linked to different support types.
+
+- hosting@altis-dxp.com 
+This linked to an Incident type (non-urgent).
+
+- support@altis-dxp.com
+This is linked to the Question support type.
+
+- sysops@altis-dxp.com 
+This linked to an ‘Urgent’ Incident type, and is linked to PagerDuty, which alerts on-call engineers to the emergency. 
```

Comment:
```suggestion
- **hosting@altis-dxp.com**<br />
   This will create an Incident type ticket (non-urgent).
- **support@altis-dxp.com**<br />
   This will create a Question type ticket.
- **sysops@altis-dxp.com**<br />
   This will create an ‘Urgent’ Incident type ticket, and is linked to PagerDuty. This will alert on-call engineers to the emergency. 
```

Some formatting and wording changes